__tests__/app.test.js

```diff
@@ -139,31 +138,83 @@ test("200: responds with updated votes", () => {
 
  test("400: responds with invalid input for a bad request", () => {
   const newVote = {
-    inc_vote: 3
+    inc_votes: 3
   }
   return request(app)
-    .patch("/api/articles/%")
+    .patch("/api/articles/biij")
     .send(newVote)
     .expect(400)
+    .then(({body})=> {
+      expect(body.msg).toBe("Bad Request")
+    })
  })
 
  test("404: responds with not found error if the article doesn't exist but the entry is valid", () => {
   const newVote = {
-    inc_vote: 3
+    inc_votes: 3
   }
   return request(app)
     .patch("/api/articles/1000")
     .send(newVote)
     .expect(404)
+    .then(({body})=> {
+      expect(body.msg).toBe("This data does not exist")
+    })
  })
+})
 
- test("500: responds with server error if the newVote does not contain the key inc_vote", () => {
-  const newVote = {
-    vote: 3
-  }
-  return request(app)
-    .patch("/api/articles/6")
-    .send(newVote)
-    .expect(500)
- })
-})
\ No newline at end of file
+
+
+describe("8. GET: /api/articles", () => {
```

Comment:
We're missing tests and functionality for the topic query in this endpoint. 
The end point should also accept the following query:
- topic, which filters the articles by the topic value specified in the query. If the query is omitted the endpoint should respond with all articles.
We'll need to also check the topic query is valid and have error handling for if it isn't